style.css

```
@@ -0,0 +1,71 @@
+header {
+	display: flex;
+	justify-content: flex-end;
+}
+a {
+    color: #222222;
```

Краще використовувати скорочений запис
```suggestion
    color: #222;
```